server/views/formPages/incident/details.html

```diff
@@ -120,11 +120,11 @@ <h1 class="govuk-heading-xl mainHeading">Use of force details</h1>
               "name": "guidingHold",
               "value": data.guidingHold,
               "options": [{
-                "value": "yes",
+                "value": "Yes",
```

Comment:
As discussed - we should probably ticket this up to store this more as a "code" rather than a "label"

Reply:
I will do this as a separate ticket because it will impact across every page in the application.